components/director/src/config.rs

```diff
@@ -98,23 +99,61 @@ impl Config {
     fn lookup_service_param(toml: &toml::Value,
                             service_name: &str,
                             param_name: &str)
-                            -> Option<String> {
+                            -> Result<Option<String>> {
```

Comment:
right, I gave it the same return type as the new function I added for consistency.

> On Jul 18, 2016, at 1:33 PM, Jamie Winsor <notifications@github.com> wrote:
> 
> In components/director/src/config.rs:
> 
> > @@ -98,23 +99,61 @@ impl Config {
> >      fn lookup_service_param(toml: &toml::Value,
> >                              service_name: &str,
> >                              param_name: &str)
> > -                            -> Option<String> {
> > +                            -> Result<Option<String>> {
> This funciton never returns an Err case so just using Option is fine here
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>